src/python/WMQuality/Emulators/DBSClient/MockDbsApi.py

```diff
@@ -88,6 +88,8 @@ def listFileLumiArray(self, **kwargs):
         if 'logical_file_name' in kwargs and len(kwargs['logical_file_name']) > 1:
             origArgs = copy.deepcopy(kwargs)
             returnDicts = []
+            if isinstance(kwargs['logical_file_name'], str):
+                kwargs['logical_file_name'] = [kwargs['logical_file_name']]
```

Comment:
an additional comment would be welcome here. Why do you convert it to the list? And, again code is not protected if kwargs will not have such key in it.